fastlane/Fastfile

```diff
@@ -101,10 +101,10 @@ platform :ios do
       set_github_release(
           repository_name: "rakutentech/ios-miniapp",
           api_token: ENV["GITHUB_API_TOKEN"],
-          name: "Miniapp",
+          name: "Miniapp SDK v"+File.read("../tmp/version.env"),
```

Comment:
I'm not sure we need a name to be honest, when releasing manually the release takes the tag as a name

Reply:
So we can get rid of the name and just tag is enough?

Reply:
I think so, not tested. But don't worry we can also keep it like this